UPGRADE-5.0.md

```diff
@@ -54,6 +54,7 @@ HttpFoundation
 
  * The `$size` argument of the `UploadedFile` constructor has been removed.
  * The `getClientSize()` method of the `UploadedFile` class has been removed.
+ * The `getSession()` method of the `Request` class throws an exception when session is null.
```

Comment:
mentioning the deprecation in UPGRADE-4.1.md and HttpFoundation/CHANGELOG.md is missing

Reply:
Done. I will send a PR to update the Github PR template to help contributors to do such things, thanks.